pgoapi/pgoapi.py

```diff
@@ -308,6 +358,19 @@ def spin_all_forts_visible(self):
     def return_to_start(self):
         self.set_position(*self._origPosF)
 
+    def walk_to_fort(self, fort_data, directly=False):
+        fort = fort_data[0]
+        self.log.info("Walking to fort at  http://maps.google.com/maps?q=%s,%s", fort['latitude'],
+                        fort['longitude'])
+        if directly is True:
```

Comment:
Yeah, the variable is only determined at call time.  We only want to walk directly if you are in experimental, are spinning all forts, and have called `walk_to` inside of `walk_to` (i.e recursively).  We are using the `directly` kwarg to make sure it doesn't recursively try to spin all forts as well, it skips that part and just walks to the nearest fort